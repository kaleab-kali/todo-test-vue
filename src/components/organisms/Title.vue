<template>
  <TitleName />
</template>

<script>
import TitleName from "@/components/atoms/TitleName";
export default {
  components: {
    TitleName,
  },
};
</script>

<style lang="scss" module></style>
