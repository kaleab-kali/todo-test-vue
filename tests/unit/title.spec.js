import { mount } from "@vue/test-utils";
import Title from "@/components/organisms/Title.vue";

describe("Atom Title Component", () => {
  it("Correct layout", () => {
    const wrapper = mount(Title);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
