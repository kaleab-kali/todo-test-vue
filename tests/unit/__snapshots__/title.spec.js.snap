// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom Title Component Correct layout 1`] = `<header><h1 class="title">to do list</h1></header>`;
